tutorials/algorithms/01_algorithms_introduction.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
```

Comment:
<p>The primitives can be based on (double space)a simulator </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>